src/communication/network_interface_sr.c

```diff
@@ -11510,3 +11511,87 @@ smmon_disable_force (THREAD_ENTRY * thread_p, unsigned int rid, char *request, i
   css_send_data_to_client (thread_p->conn_entry, rid, reply, OR_ALIGNED_BUF_SIZE (a_reply));
 #endif // !WINDOWS
 }
+
+/*
+ * stdes_set_tran_start - set the start time and sql text of the transaction
+ *   thread_p(in): the thread pointer
+ *   rid(in): the request id
+ *   request(in): the request
+ *   reqlen(in): the request length
+ */
+void
+stdes_set_tran_start (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
+{
+  char *sql_user_text = NULL;
+  int tran_index = -1;
+
+  or_unpack_string (request, &sql_user_text);
```

Comment:
적용하시면, db_private_free_and_init (thread_p, sql_user_text) 호출 불필요 합니다.

```suggestion
  or_unpack_string_nocopy (request, &sql_user_text);
```

Reply:
[72eb044](https://github.com/CUBRID/cubrid/commit/72eb04468e25378645d7ab027a8a12917668da62)에서 반영하였습니다.